.gitignore

```diff
@@ -31,3 +31,6 @@ coverage.xml
 requirements.in
 requirements.txt
 .eggs/
+
+# Sphinx documentation
+docs/requirements.txt
```

Comment:
Instead of ignoring this file, you should unignore this entry. Also, don't do this in a repo-global gitignore but instead create a separate one in the `docs/` folder.